src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -41,42 +41,278 @@
 
 # adapted from main.snip:apiMethods(xapiClass) and its dependencies
 # For real use, we'll want to switch on the calling form id.
-@private generateSample(apiMethod, callingForm, initCode)
-  @switch callingForm
+@private generateSample(apiMethod, sample)
+  @switch sample.callingForm
   @case "Request"
-    {@syncMethodSampleCode(apiMethod, initCode)}
+    {@syncMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "RequestPaged"
-    {@pagedIterableMethodSampleCode(apiMethod, initCode)}
+    {@pagedIterableMethodSampleCode(apiMethod, sample)}
   @case "Flattened"
-    {@syncMethodSampleCode(apiMethod, initCode)}
+    {@syncMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "FlattenedPaged"
-    {@pagedCallableMethodSampleCode(apiMethod, initCode)}      
+    {@pagedCallableMethodSampleCode(apiMethod, sample)}      
   @case "Callable"
-    {@callableMethodSampleCode(apiMethod, initCode)}
+    {@callableMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "CallableList"
-    {@unpagedListCallableMethodSampleCode(apiMethod, initCode)}
+    {@unpagedListCallableMethodSampleCode(apiMethod, sample)}
   @case "CallablePaged"
-    {@pagedCallableMethodSampleCode(apiMethod, initCode)}
+    {@pagedCallableMethodSampleCode(apiMethod, sample)}
   @case "CallableStreamingBidi"
-    {@bidiStreamingCallableSampleCode(apiMethod, initCode)}      
+    {@bidiStreamingCallableSampleCode(apiMethod, sample)} 
   @case "CallableStreamingClient"
-    {@clientStreamingCallableSampleCode(apiMethod, initCode)}
+    {@clientStreamingCallableSampleCode(apiMethod, sample)}
   @case "CallableStreamingServer"
-    {@serverStreamingCallableSampleCode(apiMethod, initCode)}
+    {@serverStreamingCallableSampleCode(apiMethod, sample)}
   @case "LongRunningCallable"
-    {@asyncOperationCallableMethodSampleCode(apiMethod, initCode)}
+    {@asyncOperationCallableMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "LongRunningFlattenedAsync"
-    {@asyncOperationMethodSampleCode(apiMethod, initCode)}
+    {@asyncOperationMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @case "LongRunningRequestAsync"
-    {@asyncOperationMethodSampleCode(apiMethod, initCode)}
+    {@asyncOperationMethodSampleCode(apiMethod, sample)}
+    {@processResponse(sample)}
   @default
-    $unhandledCallingForm: {@callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
+    $unhandledCallingForm: {@sample.callingForm} in sample "{@apiMethod.getClass.getSimpleName}"$
   @end
 @end
 
+# The snippets are copied and modified from method_sample.snip since standalone
```

Comment:
"... are modified versions of those in method_sample.snip since standalone samples can perform more sophisticated response handling."